CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the homebrew cookbook.
 
 ## Unreleased
 
+- Updated library call for new homebrew class name
```

Comment:
Probably best mention the chef version requirement here.

Reply:
good point updated in latest commit